extensions/amp-story/1.0/amp-story-interactive-binary-poll.css

```diff
@@ -158,3 +158,40 @@
   .i-amphtml-story-interactive-option-percentage-text {
   transform: scale(1) !important;
 }
+
+.i-amphtml-story-interactive-binary-poll-container.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection) 
+  .i-amphtml-story-interactive-option:before {
+  background-color: var(--i-amphtml-interactive-theme-shading-flash);
+  content: '' !important;
+  position: absolute !important;
+  width: 100% !important;
+  height: 100% !important;
+  left: 0 !important;
+  opacity: 0 !important;
+  animation: i-amphtml-interactive-animation-landing-animation-flash-background var(--i-amphtml-interactive-animation-time) !important;
+  animation-delay: var(--i-amphtml-interactive-landing-animation-delay-offset-el1) !important;
+}
+
+.i-amphtml-story-interactive-binary-poll-container.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection) 
```

Comment:
WDYT about removing ".i-amphtml-story-interactive-binary-poll-container" from the rule, since it's already being targeted with the -active and would make it more readable (here and other files)?